tensorflow/core/kernels/data/shuffle_dataset_op_test.cc

```diff
@@ -0,0 +1,665 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Do you plan to add tests for the ShuffleAndRepeat op in a separate PR?

Reply:
Yes, I was planning to add another file `shuffle_and_repeat_dataset_op_test.cc` to test for the ShuffleAndRepeat op in a separate PR. But it looks like that it will share a lot of code with this PR, it may be better to add the tests for `ShuffleAndRepeat` in the same file. Which way do you prefer to?

Reply:
SGTM

Reply:
fixed via https://github.com/tensorflow/tensorflow/pull/27969/commits/bccf25812b0dbe50aaa06d06b5c8c1c22dd9c371